amundsen_application/static/js/components/common/Breadcrumb/styles.scss

```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+
+.container.amundsen-breadcrumb {
+  margin-top: 13px;
+  margin-bottom: -32px;
+}
```

Comment:
Would prefer to nest the breadcrumb inside the regular container with the rest of the content with:

```
.amundsen-breadcrumb {
  margin-top: -35px;
  margin-bottom: 16px;
}
```
Basically `margin top + margin bottom + height == 0` and the element will have no affect on the flow of elements below it.

Reply:
I also don't mind taking this since you're working on other projects.

Reply:
I'll file a ticket for breadcrumbs and put it on the tabledetail page as well.